plugins/GSdx/GSState.cpp

```diff
@@ -46,7 +46,7 @@ GSState::GSState()
 	, m_frameskip(0)
 	, m_crcinited(false)
 {
-	m_nativeres = !!theApp.GetConfig("nativeres", 1);
+	m_nativeres = !!theApp.GetConfig("upscale_multiplier",1)==1 ? 1:0;
```

Comment:
i think the default value (the 1 after the comma) is only if it doesn't exist already, otherwise it'll return the true value